cspell.json

```
@@ -402,7 +401,8 @@
         "\\[`\\w+`\\]",
         "ve{2,}r{2,}y",
         "ve+r+y+long\\w*",
-        "\\/.*?\\/"
+        "\\/.*?\\/",
+        "\"?author.*\"?: \".*\""
```

```suggestion
        "author: \".*?\"",
        "authorURL: \".*?\"",
        "\"author\": \".*?\""
```

Let's use separate regrex, more clear